Dockerfile.armhf

```diff
@@ -17,6 +17,10 @@
 
 FROM armhf/debian:jessie
 
+# Allow replacing httpredir mirror
+ARG APT_MIRROR=httpredir.debian.org
+RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
```

Comment:
@andrewhsu how about using the same here as https://github.com/docker/docker/blob/75091abe76531d4373aba2da680b695e5dbb8fc8/contrib/builder/deb/amd64/debian-jessie/Dockerfile#L7-L9 ?